app/src/main/java/org/mozilla/focus/browser/VideoVoiceCommandMediaSession.kt

```diff
@@ -157,6 +161,14 @@ class VideoVoiceCommandMediaSession @UiThread constructor(
         mediaSession.release()
     }
 
+    /**
+     * Potentially handles key events before they are received by the system for standard handling
```

Comment:
Good comments here.

Reply:
I've been spending as much time writing the comments as the code, I think. :P